src/test/java/seedu/address/logic/commands/RemoveCommandTest.java

```diff
@@ -44,4 +47,17 @@ public void execute_itemAcceptedByModel_removeSuccessful() throws CommandExcepti
                 commandResult.getFeedbackToUser());
     }
 
+    @Test
+    public void equals() {
```

Comment:
I believe you should also try a test case with a non RemoveCommand object. I believe the current equals method doesn't handle this and will throw a ClassCastException